src/main/java/de/bonndan/nivio/output/map/svg/SVGItem.java

```diff
@@ -81,11 +81,11 @@ public DomContent render() {
             circle.attr("stroke-dasharray", 5);
             circle.attr("opacity", 0.7);
         }
-        ContainerTag inner = SvgTagCreator.g(circle, content, children)
-                .attr("class", "hexagon");
+        ContainerTag inner = SvgTagCreator.g(circle, content, children);
 
         return SvgTagCreator.g(inner, icon)
-                .attr("class", "hexagon-group")
+                .attr("data-identifier", this.id)
+                .attr("class", "item")
```

Comment:
What does the data-identifier and the class attribute change here?

Reply:
The element gets on onClick event (all ".item" elements) and the identifier is used to show correct item data in a dialogue.

Reply:
Okay, I haven't looked into the frontend logic querying for these yet.